README.md

```diff
@@ -0,0 +1,14 @@
+# Cruise Ships
+This project is part of my learning about Object Oriented Programming at Command Shift.
+
+## Description
+The concepts I've learnt throughout this project are:
+
+- Domain Modelling
+- Object-oriented Programming
+- Single Responsibility Principle
+- Dependency inversion
+- Mocks
+- Spies
+
+To see the end result of this project, please see https://github.com/josenymad/cruise-ships-gui 
```

Comment:
Id rephrase this a little. That repo isnt the end result of this project, this is the foundation for that one, it would be better to describe this in its own right here and explain how its installed, tested and used as its own standalone thing. By all means reference the new project, but including these other things here would be more useful